docs: create a FAQ-style page for platform setup (#15425)
istio/istio.io: content/en/docs/setup/platform-setup/amazon-eks/amazon-eks.md

---
title: Amazon EKS
description: Instructions to set up Istio on Amazon EKS in AWS cloud.
weight: 7
skip_seealso: true
keywords: [platform-setup,aws-cloud,eks]
owner: istio/wg-environments-maintainers
test: n/a
---

> Follow [EKS Blueprints for Istio](https://aws-ia.github.io/terraform-aws-eks-blueprints/patterns/istio/) instructions to provision EKS cluster with Istio setup in AWS cloud.

Reviewer: ```suggestion
Follow [Istio on EKS](https://github.com/aws-samples/istio-on-eks) instructions to provision EKS cluster with Istio setup in AWS cloud.
```
